pkg/cmd/attestation/attestation.go

```diff
@@ -17,9 +17,9 @@ func NewCmdAttestation(f *cmdutil.Factory) *cobra.Command {
 		Short:   "Work with artifact attestations",
 		Aliases: []string{"at"},
 		Long: heredoc.Doc(`
-			# NOTE: This feature is currently in beta, and subject to change.
+### NOTE: This feature is currently in beta, and subject to change.
 
-			Download and verify artifact attestations.
+Download and verify artifact attestations.
```

Comment:
There `heredoc` library should remove all left hand whitespace from the `go` file in final rendering, so no need to left hand align it.